src/Components/Horizontal_Card/HorizontalCard.jsx

```diff
@@ -1,30 +1,95 @@
-import React from 'react';
+import { useState, useEffect } from 'react';
 import { Link } from 'react-router-dom';
+import { useAuth, useCart } from '../../Context';
 import styles from './HorizontalCard.module.css';
-import { Fifa } from '../../Assets/images';
+import axios from 'axios';
+
+const HorizontalCard = ({ _id, title, image, price, qty }) => {
+	const [qtyChangeLoader, setQtyChangeLoader] = useState(false);
+
+	const { cartState, cartDispatch } = useCart();
+	const { authState } = useAuth();
+
+	useEffect(() => {
+		return () => {};
+	}, []);
+
```

Comment:
You can remove this useEffect if you have no plans to use it.

Reply:
This was used to avoid a memory leak bug, I have to research more about it before I remove it